dist/infragram.js

```diff
@@ -1498,9 +1454,10 @@ function _slicedToArray(arr, i) { if (Array.isArray(arr)) { return arr; } else i
         Presets = require('../ui/presets')(options, save_infragrammar_inputs);
         Analysis = require('../ui/analysis')(options, save_infragrammar_inputs);
         Colorize = require('../ui/colorize')(options);
+        Saving = require('../ui/saving')(options);
```

Comment:
This can be removed, as mentioned below it was removed in https://github.com/publiclab/infragram/pull/268